### DISCUSSION TITLE
v2.20.4

### DISCUSSION THREAD

**Discussion by tyrasd** [openstreetmap/iD#8942] [Announcements]
#### :bug: Bugfixes
* Fix rendering of HTML content in ~~KeepRight and~~ ImproveOSM issues ([#8928])
* Fix "local storage is exceeded" error message showing up multiple times ([#8930])

[#8928]: https://github.com/openstreetmap/iD/pull/8928
[#8930]: https://github.com/openstreetmap/iD/pull/8930

<hr /><em>This discussion was created from the release <a href='https://github.com/openstreetmap/iD/releases/tag/v2.20.4'>v2.20.4</a>.</em>

**Comment by sekerob** [comment]
I noticed from previous release 2.20.3 that the red bar at the bottom telling that too many edits were made to save has become even more repulsive than it was before at single line. Now with every few clicks (to finish the task at hand, the line starts replicating and gradually filling the screen bottoms up to the point that I can't continue for lack of view on the area to complete. If you want volunteers to take a break from mapping, please do tell.

Edit: Cleared the browser caches, Java buffers, cookies, exited the browser, reloaded, signed in an made small edits, STILLl 2.20.3

**Reply by tyrasd** [reply]
Hey. I'm really sorry for adding the bug. :bow: It snuck in a security related bugfix (#8813), in case you're curious. `v2.20.4` is prepared as a hotfix release, but still needs to be deployed on osm.org – see https://github.com/openstreetmap/openstreetmap-website/pull/3446 for that. I'm sure it will be made available very soon. In the meantime you can already use https://ideditor-release.netlify.app/ to map with iD v2.20.4.

**Reply by sekerob** [reply]
Hi, Using your 'live' release ATM. I would seriously reconsider this edit screen real estate eating line. We have right top the Save button that incrementally changes from white, to yellow to orange and red (100 changes I think). I'd propose you turn this button Maroon or another obnoxious colour, maybe a blinking attribute (which some will surely have an issue with). It will be even more noticeable if going from yellow to maroon as the criteria for the 'too many edits' seem to be different, seen it appear when still in yellow state. Maybe move the warning to bottom of the left hand sidebar as an overlay. Anyway, Vivaldi is my browser and unfortunately it does not have the Firefox equivalent of about:config to increase the edit buffer.

My 2 cents

**Reply by tyrasd** [reply]
That's a good suggestion, I will definitely consider it. :+1: //edit: I've opened #8950 for it
